Userland/Utilities/run-tests.cpp

```diff
@@ -280,9 +280,7 @@ FileResult TestRunner::run_test_file(ByteString const& test_path)
             break; // we'll end up with a failure
 
         if (WIFEXITED(wstatus)) {
-            if (wstatus == 0) {
-                test_result = Test::Result::Pass;
-            }
```

Comment:
eh? Why remove this? if anything we should be checking `if (WEXITSTATUS(wstatus) == 0)`

Reply:
big blunder. fixed.